app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/livecamera/ClientLiveCameraFragment.kt

```diff
@@ -44,11 +44,12 @@ class ClientLiveCameraFragment : BaseDaggerFragment(), ServiceConnection {
     private var socketBinder: WebSocketClientService.Binder? = null
     private var webRtcBinder: WebRtcService.Binder? = null
 
-    private var errorOccurs = false
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        viewModel.selectedChildAvailability.observe(this, Observer { it?.let(::onAvailabilityChange) })
+        viewModel.selectedChildAvailability.observe(
+            this,
+            Observer { it?.let(::onAvailabilityChange) })
         viewModel.showBackButton(true)
         requireContext().apply {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/59ecd15255bdd529a391ea0d5072dc37d88e9138/app/src/main/kotlin/co/netguru/baby/monitor/client/feature/client/home/livecamera/ClientLiveCameraFragment.kt#L54"></span>apply expression can be omitted</td>
    </tr>
  </tbody>
</table>